chromoscores/scorefunctions.py

```diff
@@ -123,38 +188,52 @@ def peak_score(
 """Isolation score"""
 
 
-def isolation_score(contact_map, delta, diag_offset, max_dist, state, pseudo_count=1):
+def isolation_score(snippet, delta, diag_offset, max_dist, state, pseudo_count=1):
+    """
+    parameters
+    ----------
+    snippet: snippet of the contact map around the boundary element
+    delta: distance from the border between in_tad and out_tad. It is defined to exclude
+           flames when extracting in_tad and out_tad areas.
+    diag_offset: distance from the diagonal. This also determines the size of the snippet.
+    max_distance: maximum distance from the diagonal
+    state: 1 for triangle snippets, 0 for square snippets
+    pseudo_count: pseudo count to avoid division by zero
+
+    returns
+    -------
+    ratio of the mean of the area inside tads and the area outside tad
+
+    """
     in_tad, out_tad, pile_center = get_isolation_snippets(
-        contact_map, delta, diag_offset, max_dist, state
+        snippet, delta, diag_offset, max_dist, state
     )
     assert pile_center.shape == (len(in_tad), len(in_tad))
     return (pseudo_count + np.mean(pile_center[in_tad > 0])) / (
         pseudo_count + np.mean(pile_center[out_tad > 0])
     )
 
 
-"""Tad score"""
 
 
-def tad_score(contact_map, stall_list, index, delta, diag_offset, max_dist):
+def tad_score(contact_map, boundary_list, index, delta, diag_offset, max_dist):
```

Comment:
this is probably better to be deleted now that we know the issues with this score